joda-money/src/main/java/com/fasterxml/jackson/datatype/jodamoney/AmountConverter.java

```diff
@@ -0,0 +1,13 @@
+package com.fasterxml.jackson.datatype.jodamoney;
+
+import org.joda.money.CurrencyUnit;
+import org.joda.money.Money;
+
+import java.math.BigDecimal;
+
+public interface AmountConverter<T> {
```

Comment:
Just realized that type reference here is not very valuable since the common base type is `java.lang.Object`.
May be left but wondering if it'd make sense to just drop it.

Reply:
my intention was to make the "output" type somehow prominent, but it's true that doesn't bring much value - will remove